internal/npm_install/npm_install.bzl

```diff
@@ -26,6 +26,10 @@ load("//internal/common:os_name.bzl", "is_windows_os")
 load("//internal/node:node_labels.bzl", "get_node_label", "get_npm_label", "get_yarn_label")
 
 COMMON_ATTRIBUTES = dict(dict(), **{
+    "timeout": attr.int(
+        default = 3600,
+        doc = """Maximum duration of the command "yarn install" in seconds.""",
```

Comment:
nit: this doc is wrong, applies to npm install too

Reply:
Thanks. Generalized to "Maximum duration of the package manager execution in seconds."